modules/reactor/doc/content/source/meshgenerators/PinMeshGenerator.md

```diff
@@ -0,0 +1,41 @@
+# PinMeshGenerator
+
+!syntax description /Mesh/PinMeshGenerator
+
+## Overview
+
+This object is designed to be used in the Reactor MeshGenerator workflow, which also consists of [`ReactorMeshParams`](ReactorMeshParams.md), [`AssemblyMeshGenerator`](AssemblyMeshGenerator.md), and [`CoreMeshGenerator`](CoreMeshGenerator.md).
+
+The `PinMeshGenerator` object generates square or hexagonal reactor geometry pin cell structures which may be combined into larger assembly structures using `AssemblyMeshGenerator`. The block IDs, external boundary ID, region IDs (e.g., materials), and reporting IDs (extra element integers identifying unique planes and pins) are automatically assigned once the user provides some basic information. The pin defined may be extruded to three dimensions ([!param](/Mesh/PinMeshGenerator/extrude) equals 'true'), however it cannot be used in further mesh definition with `AssemblyMeshGenerator`.
```

Comment:
separate the paragraph to talk about extrusion
the sentence is confusing right now.
It's however, if it s extruded, it cant be used

Reply:
Reworded